pkg/handlers/s3_handler.go

```diff
@@ -156,11 +157,49 @@ func (h *s3Handler) OnAdd(ns string, vc kvcv1.VolumeConfig, controllerRef metav1
 	nodeLabelKey := fmt.Sprintf("%s/%s-%s-%s", kvcv1.GroupName, ns, controllerRef.Name, vc.ID)
 	for _, kvcName := range kvcNames {
 		err := waitForPodSuccess(podClient, kvcName, ns, timeout)
+
 		if err != nil {
+			downloadErrMsg := "error during data download and failed to fetch logs for pod"
```

Comment:
Lets make this just `error during data download`? The whole message looks awkward.